### ISSUE TITLE
stable/percona-xtradb-cluster bootsrap cluster

### ISSUE THREAD

**Issue by yassinecharjami** [helm/charts#11532]
**Is this a request for help?**:
Yes
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug report

I'm not sure if the cluster is bootstrapped or not while starting because i checked grastate.dat of all pods i didn't find safe_to_bootstrap = 1 ( always 0 ), also i checked the variable wsrep_cluster_address i didn't find the IPs of other pods of the cluster. 
The main problem is when i shutdown 2 pods in a cluster of 3 pods i can't restart them from the third.
Please, is there any solution to restart the cluster properly ?

Thanks

**Comment by derekyle** [comment]
I ran into this same issue. The only solution I found was to modify the entrypoint script to change safe_to_bootstrap = 1 manually. 

And there is an inherent limitation with this method and that is that a statefulset launches the replicas in order. So you will only have the choice to set the first pod "...-0" as safe to bootstrap. Because if you don't, then the remaining pods will never launch. So if a failure caused an issue and it just happens that your first pod is not the most advanced one, you have a problem.
